packages/notifi-react-card/lib/components/intercom/NotifiIntercomCard.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import {
+  NotifiSubscriptionContextProvider,
+  useNotifiClientContext,
+} from '../../context';
+import type { ErrorStateCardProps } from './ErrorStateCard';
+import type { LoadingStateCardProps } from './LoadingStateCard';
+import { NotifiIntercomCardContainer } from './NotifiIntercomCardContainer';
+import type { NotifiStartChatButtonProps } from './NotifiStartChatButton';
+
+export type NotifiInputSeparators = {
```

Comment:
i think we already have these types, no?

Reply:
ah, you are right, removed and import from subscritionCard